### ISSUE TITLE
[1.5.3] New version CANNOT be start.

### ISSUE THREAD

**Issue by manesec** [AllTheMods/ATM-6#1145]
**Describe the bug**
I am download the version 1.5.3 in [curseforge](https://www.curseforge.com/minecraft/modpacks/all-the-mods-6/files/3221291) and import it in MultiMC, when I run , It show an error.

**To Reproduce**
1. Download it 1.5.3
2. import in MultiMC
3. run and show bug

**Screenshots**
https://i.loli.net/2021/03/01/yiEzn6FXNwjgM3S.png

**Additional context**

**Logs**
https://gist.github.com/manesec/97a3eaadd40896af5ec790c032e64f0f

**Comment by NetMasterFlex** [comment]
I recently imported and installed ATM6 Version 1.5.3.  I downloaded from Curseforge , imported into Multi-MC,and joined my personal server that I’ve been updating since 10 to 15 versions back.  I had no issues.  

Have you tried re-downloading the file. Also make sure you are running the latest version of Multi-MC. The issue doesn’t appear to be something on the ATM side of situation. I just wanted to let you know that 1.5.3 is working. 

Details of the error would be helpful as well, maybe provide a screenshot. Also share your system specs, memory allocated to the game etc.  Java version? Etc.  

It’s hard for anyone to help if little information is provided.
